plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-terms-block/editor.scss

```diff
@@ -1,6 +1,16 @@
+.edit-post-visual-editor {
+
+	.is-small,
+	.is-medium {
+		.wc-block-checkout__terms {
```

Comment:
Why are we applying `padding: 4px;` to the Terms and Conditions block in the editor?

Reply:
This is so it can be selected more easily and the borders on selected blocks is more visible.

Reply:
Thanks for clarifying this, and thanks for showing me this during a recent call.